src/Startup.cpp

```diff
@@ -457,6 +461,10 @@ Startup()
 #endif
 #endif
 
+#ifdef USE_GDI
+  GdiStartup();
```

Comment:
This needs to be called in all programs using the UI toolkit library - i.e. all those programs in `test/src/`. You need a better approach for this, e.g. in `class ScreenGlobalInit` which is used by all relevant programs.

Reply:
Ok, I forgot this move to  class ScreenGlobalInit  - your way is much better!